apps/login-flow-e2e/src/integration/MenuApp.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { i18nKeys, initI18Next } from '@form-exercise/i18n';
+import { t } from '../support/app.po';
+
+initI18Next();
+
+describe('MenuApp', () => {
+  beforeEach(() => {
+    cy.logout();
+  });
+
+  it('Test if there is no menu list when user is not logged in', () => {
+    cy.visit('/');
+    cy.get('.MuiToolbar-root').children('ul').should('not.exist');
+  });
+  it('Test if there is a menu when user is logged in ', () => {
+    cy.login();
+    cy.visit('/admin');
+    cy.get('.MuiToolbar-root').children('ul').children('li').should('exist');
```

Comment:
it would be better to check each menu title exists and is a link to a different page, this test (and the previous cy.get too) is too vague and could lead to false positives in my opinion.

In the same way you check a button text (with i18n keys and t function), you should check text and link to other pages are visually present on the page.